src/realm-settings/TokensTab.tsx

```diff
@@ -70,36 +69,15 @@ export const RealmSettingsTokensTab = ({
     javaKeystoreAlgOptions!
   );
 
-  const form = useForm<RealmRepresentation>();
-  const { control } = useFormContext();
+  const form = useFormContext<RealmRepresentation>();
+  const control = form.control;
```

Comment:
You can use destructuring to shorten this statement. In fact we might actually want to create an ESLint rule for this (that's for another time).

```suggestion
  const { control } = form;
```